cmd/checkMigrations.go

```diff
@@ -114,7 +114,7 @@ func getGithubFileNames() ([]string, error) {
 	jsonErr := json.Unmarshal(body, &files)
 
 	if jsonErr != nil {
-		return []string{}, fmt.Errorf(errorContext, jsonErr)
+		return []string{}, fmt.Errorf("Failed to unmarshal %s, %w", string(body), jsonErr)
```

Comment:
super minor but I think technically we want to avoid capitalization in error messages: https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
Bah humbug. Fixed.